app/screens/channel_bookmark/components/bookmark_link.tsx

```diff
@@ -121,7 +129,7 @@ const BookmarkLink = ({disabled, initialUrl = '', resetBookmark, setBookmark}: P
             <View style={descContainer}>
                 <FormattedText
                     id='channel_bookmark_add.link.input.description'
-                    defaultMessage='Add a link to any post, file, or any external link'
+                    defaultMessage='Add a link to any post, file, or any external link. You can also use app links like mattermost://'
```

Comment:
If we change this, we also have to change the `en.json` file for this to be reflected in the app.

Reply:
@larkox Good catch! I've updated the `en.json` file to include the new description text that mentions app links support. The localization string `channel_bookmark_add.link.input.description` now properly reflects the updated functionality.

Note: Since the server-side changes (https://github.com/mattermost/mattermost/pull/30956) are still in Open status and not merged yet, this mobile implementation is ready but will require the server-side support to be fully functional.

Reply:
What will happen for old servers (servers without the change)? We need to make sure the app still works (and we don't provide misleading information) if the server doesn't support this.